### PR TITLE
[13.0][MIG] procurement_purchase_no_grouping: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT21639

### CODE REVIEW

File: procurement_purchase_no_grouping/models/stock_rule.py [hunk 1]
```diff
@@ -3,23 +3,20 @@
...
+            ] = procurement.product_id.categ_id.procured_purchase_grouping
+        return super()._run_buy(procurements)
 
-    def _make_po_get_domain(self, values, partner):
-        if self.env.context.get("grouping", "standard") == "order":
-            return (("id", "=", 0),)
-        return super()._make_po_get_domain(values, partner)
+    def _make_po_get_domain(self, company_id, values, partner):
+        if values.get("grouping") == "order":
+            return (("id", "=", 0), ("company_id", "=", company_id.id))
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/models/stock_rule.py]: Why adding `company_id` to the domain? The result is going to be the same: 0 records.

File: procurement_purchase_no_grouping/models/purchase_order.py [hunk 2]
```diff
@@ -9,12 +9,27 @@
...
+        product_id,
+        product_qty,
+        product_uom,
+        location_id,
+        name,
+        origin,
+        company_id,
+        values,
     ):
         grouping = product_id.categ_id.procured_purchase_grouping
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/models/purchase_order.py]: Isn't `grouping` in `values` for not having to access again in the records?

File: procurement_purchase_no_grouping/tests/test_procurement_purchase_no_grouping.py [hunk 3]
```diff
@@ -32,15 +32,27 @@ def setUpClass(cls):
...
-            self.product,
-            1,
-            self.product.uom_id,
-            self.location,
-            False,
-            self.origin,
-            self.values,
+        procurement_group_obj = self.env["procurement.group"]
+        procurements = []
+        procurements.append(
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/tests/test_procurement_purchase_no_grouping.py]: Why adding a list if there's only one element?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: procurement_purchase_no_grouping/models/stock_rule.py (hunk 1 above)
Comment (Tardo) [reply procurement_purchase_no_grouping/models/stock_rule.py]: Oops!

**Comment by Tardo** [comment]
Changes done! thanks.

**Review by pedrobaeza — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]
Functional review + code review

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**